### PR TITLE
Handle some for obvious mistakes more user friendly

### PR DESCRIPTION
**Pull request by rhafer**

### CODE REVIEW

**Comment by rhafer** [comment]
I think we should have a unit test at least for the 2nd commit. Still looking into that.

File: psql2mysql.py [hunk 1]
```diff
@@ -181,10 +189,10 @@ def migrate(self):
         target_tables = self.target_db.getTables()
 
         if not source_tables:
```
Comment (jsuchome) [inline psql2mysql.py]: Shouldn't this check be part of `precheck` as well?

**Comment by jsuchome** [comment]
Could you add something like this as extra commit? I think the success case should print some OK message:

```
diff --git a/psql2mysql.py b/psql2mysql.py
index be0286a..7386bcc 100644
--- a/psql2mysql.py
+++ b/psql2mysql.py
@@ -244,9 +244,11 @@ def do_prechecks(config):
     db = DbWrapper(cfg.CONF.source)
     db.connect()
     tables = db.getSortedTables()
+    precheck_ok = True
     for table in tables:
         incompatibles = db.scanTablefor4ByteUtf8Char(table)
         if incompatibles:
+            precheck_ok = False
             print("Table '%s' contains 4 Byte UTF8 characters which are "
                   "incompatible with the 'utf8' encoding used by MariaDB" %
                   table.name)
@@ -269,6 +271,7 @@ def do_prechecks(config):
 
         long_values = db.scanTableForLongTexts(table)
         if long_values:
+            precheck_ok = False
             print("Table '%s' contains TEXT values that are more than %s "
                   "characters long. This is incompatible with MariaDB setup.",
                   table.name, MAX_TEXT_LEN)
@@ -287,6 +290,9 @@ def do_prechecks(config):
             print("Error during prechecks. "
                   "Too long text values found in the source database.")
 
+    if precheck_ok:
+        print("Success. No errors found during the prechecks.")
+
 
 def do_migration(config):
     migrator = DbDataMigrator(config)
```

File: psql2mysql.py (hunk 1 above)
Comment (rhafer) [reply psql2mysql.py]: What should be the outcome there? It's not exactly an error (not even handled as that in the migrate phrase). Just print a warning?

Comment (jsuchome) [reply psql2mysql.py]: I do not have the setup to test right now. So how does it behave when you run a precheck on empty table? If it does not print an ugly error, I'm fine with it.

Comment (jsuchome) [reply psql2mysql.py]: I mean - we do not have to warn during precheck that there's an empty table that should not be migrated. My only concern is that passing empty table to precheck does not break

Comment (jsuchome) [reply psql2mysql.py]: OK, if precheck does fine, that's good then.

**Pull request merged**